test/Microsoft.AspNet.TestHost.Tests/RequestBuilderTests.cs

```diff
@@ -25,7 +26,7 @@ public void AddRequestHeader()
         [Fact]
         public void AddContentHeaders()
         {
-            var server = TestServer.Create(app => { });
+            var server = new WebApplicationBuilder().Configure(app => { }).BuildTestServer();
```

Comment:
I still think we should have
`TestServer.Create(host => host.Configure(app => {}));`